build.gradle

```diff
@@ -7,8 +7,8 @@ plugins {
 apply plugin: 'java'
 apply from: 'release.gradle'
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
```

Comment:
have we finally decided to move client to the 1.8 version?

Reply:
What do I need to do to get the final decision?

Reply:
OK, I think we answered on that on today's sync-up.